packages/venia-ui/lib/components/CategoryList/categoryList.js

```diff
@@ -47,13 +43,16 @@ const CategoryList = props => {
     ) : null;
 
     let child;
+
+    // TODO: Actually handle errors; the logic below will never allow this to render
```

Comment:
Nice catch - do you mind creating a ticket for this?

Reply:
[PWA-990](https://jira.corp.magento.com/browse/PWA-990) 👍